torch/linalg/__init__.py

```diff
@@ -482,7 +482,8 @@
         - **solution** (*Tensor*): the least squares solution
         - **residuals** (*Tensor*):  if :math:`m > n` then for full rank matrices in :attr:`input` the tensor encodes
             the squared residuals of the solutions, that is :math:`||\text{input} @ x - b||_F^2`.
-            If :math:`m \le n`, an empty tensor is returned instead.
+            If :math:`m \le n` and rank is not equal to :math:`n`, an empty tensor is returned instead.
+            For the batched :attr:`input` if rank is less than :math:`n`, then a tensor filled with ``inf`` values is returned.
```

Comment:
"If attr:\`input\` is a batch of matrices and..."

Is the tensor filled with infs everywhere even if this property is true for only one matrix pairing?